docs/demo.md

```diff
@@ -1,114 +0,0 @@
----
```

Comment:
Suggestion: Should we maybe provide a placeholder that sends users to the new website? I imagine a bunch of links point to this site.